nise/generators/gcp/cloud_storage_generator.py

```diff
@@ -28,26 +27,13 @@ class CloudStorageGenerator(GCPGenerator):
 
     def _update_data(self, row):
         """Update a data row with storage values."""
-        if self.attributes:
-            row["Line Item"] = self.attributes["Line Item"]
-            row["Measurement1"] = self.attributes["Measurement1"]
-            row["Measurement1 Total Consumption"] = self.attributes["Measurement1 Total Consumption"]
-            row["Measurement1 Units"] = self.attributes["Measurement1 Units"]
-            row["Cost"] = self.attributes["Cost"]
-            row["Currency"] = self.attributes["Currency"]
-            row["Description"] = self.attributes["Description"]
-            row["Credit1"] = self.attributes["Credit1"]
-            row["Credit1 Amount"] = self.attributes["Credit1 Amount"]
-            row["Credit1 Currency"] = self.attributes["Credit1 Currency"]
-        else:
-            storage = choice(self.STORAGE)
-            row["Line Item"] = storage[0]
-            row["Measurement1"] = storage[0]
-            row["Measurement1 Total Consumption"] = self.fake.pyint()
-            row["Measurement1 Units"] = storage[1]
-            row["Cost"] = self.fake.pyint()
-            row["Currency"] = "USD"
-            row["Description"] = storage[2]
+        # if self.attributes:
```

Comment:
I think we should delete these comments and leave a todo comment saying we need to complete the storage generator. Probably need to create a jira issue too for the nise storage generator.

Reply:
addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff